src/index.js

```diff
@@ -1,38 +1,35 @@
 import './style.css';
+import CreateList from './createList.js';
 
-const taskArray = localStorage.getItem('tasks') ? JSON.parse(localStorage.getItem('tasks')) : [];
-const inputList = document.getElementById('input-List');
+const input = document.getElementById('input-List');
 
-const createItem = (item = {
-  task: inputList.value,
-  index: taskArray.length,
-  completed: false,
-}) => {
-  taskArray.push(item);
-  localStorage.setItem('tasks', JSON.stringify(taskArray));
-  window.location.reload();
-};
-
-inputList.value = '';
-inputList.addEventListener('keydown', (event) => {
-  if (event.key === 'Enter') {
-    createItem(inputList.value);
+input.addEventListener('keydown', (event) => {
+  if (event.key === 'Enter' && input.value !== '') {
+    new CreateList().create(input.value);
+    input.value = '';
   }
 });
 
-function displayItems() {
-  let tasks = '';
-  for (let i = 0; i < taskArray.length; i += 1) {
-    tasks += ` <div class="item" >
-    <div class="input-Content">
-    <div>
-        <input type="checkbox" class="complete" name="completed" />
```

Comment:
Hey 👋🏽  the `checkbox` is working perfectly but the value is not getting persisted to `localStorage` therefore the `checked` state goes away when the page is refreshed. ♻️